company/model.py

```diff
@@ -0,0 +1,88 @@
+import sqlite3
+from os import path
+
```

Comment:
Тут явно не хватает и этих импортов. Я их добавил, потому что без них некоторые перемены были не определены и программа не запускалась. Вы должны были это заметить, когда пытались ее запустить.
```suggestion
import user_input as usin
from company import views
from company.logger import make_log
from company.sql_tables import SQL_COLS
```